server/interfaces/interfaces.go

```diff
@@ -238,6 +239,74 @@ type WorkflowService interface {
 	ServeHTTP(w http.ResponseWriter, r *http.Request)
 }
 
+type GitProviders []GitProvider
+
```

Comment:
nit: throughout your change -- can you please add a newline between the code and following comment on the next line?

Reply:
Done